test/e2e/tests/e2e_test.go

```diff
@@ -164,7 +164,7 @@ func loadEnv(t *testing.T) (types.Testnet, netconf.Network) {
 	}
 	require.NoError(t, err)
 
-	testnet, err := app.TestnetFromManifest(m, manifestFile, ifd, nil)
+	testnet, err := app.TestnetFromManifest(m, manifestFile, ifd, nil, "main")
```

Comment:
we always want to use `main` here?

Reply:
This is a test, and the tag is not even used.